pr_watch/models.py

```diff
@@ -103,8 +103,8 @@ class WatchedPullRequest(models.Model):
     # TODO: Remove parameters from 'update_instance_from_pr'; make it fetch PR details from the
     # api (including the head commit sha hash, which does not require a separate API call as
     # is currently used.)
-    branch_name = models.CharField(max_length=50, default='master')
-    ref_type = models.CharField(max_length=50, default='heads')
+    branch_name = models.CharField(max_length=255, default='master')
+    ref_type = models.CharField(max_length=255, default='heads')
```

Comment:
This has only three possible values: `"repository"`, `"branch"` or `"tag"`, so there is no real need to reserve more space in the DB for this.

Reply:
Ah, good call.